backend/app/service/website_utils.go

```diff
@@ -1037,22 +1036,7 @@ func UpdateSSLConfig(websiteSSL model.WebsiteSSL) error {
 			return buserr.WithErr(constant.ErrSSLApply, err)
 		}
 	}
-	enable, auto, sslID := GetSystemSSL()
-	if enable && sslID == websiteSSL.ID {
-		fileOp := files.NewFileOp()
-		secretDir := path.Join(global.CONF.System.BaseDir, "1panel/secret")
-		if err := fileOp.WriteFile(path.Join(secretDir, "server.crt"), strings.NewReader(websiteSSL.Pem), 0600); err != nil {
-			global.LOG.Errorf("Failed to update the SSL certificate File for 1Panel System domain [%s] , err:%s", websiteSSL.PrimaryDomain, err.Error())
-			return err
-		}
-		if err := fileOp.WriteFile(path.Join(secretDir, "server.key"), strings.NewReader(websiteSSL.PrivateKey), 0600); err != nil {
-			global.LOG.Errorf("Failed to update the SSL certificate for 1Panel System domain [%s] , err:%s", websiteSSL.PrimaryDomain, err.Error())
-			return err
-		}
-		if auto {
-			_, _ = cmd.Exec("systemctl restart 1panel.service")
-		}
-	}
+	reloadSystemSSL(&websiteSSL, nil)
 	return nil
 }
 
```

Comment:
This is not a code snippet, but my suggestion would be to format both functions and variables consistently by using comments or indents.

The `saveCertificateFile` function seems like it needs some minor modifications to fix bugs:

Function name should end with 'func' as per standard Go naming conventions.

If you need help on implementing custom logic inside this function based on provided information, here's an example implementation considering those guidelines:

```go
// Function saves the certificate of the provided websiteSSL to a local file in the given directory

func saveCertificateFile(websiteSSL model.WebsiteSSL, logger *log.Logger) error {
    // Add your saving logic here...  
    return nil // or log any errors that occurred
}
```

Remember, the specific changes needed will depend heavily upon what exactly was intended by each coder, so please provide more detailed context about which parts may require modification.